mkt/__init__.py

```diff
@@ -1,18 +1,19 @@
+# flake8: noqa
```

Comment:
I've no idea, I'll let someone else who knows more about zamboni go for that. I hope so, its just wrong.